specs/src/specs/params.md

```diff
@@ -31,9 +31,9 @@ are blocked by the `x/paramfilter` module.
 | distribution.WithdrawAddrEnabled | true | Enables delegators to withdraw funds to a different address. | True |
 | distribution.BaseProposerReward | 0 | Reward in the mint demonination for proposing a block. | True |
 | distribution.BonusProposerReward | 0 | Extra reward in the mint denomination for proposers based on the voting power included in the commit. | True |
-| gov.DepositParams.MinDeposit | 1000000000utia (1000 TIA) | Minimum deposit for a proposal to enter voting period. | True |
+| gov.DepositParams.MinDeposit | 1_000_000_000 utia (1000 TIA) | Minimum deposit for a proposal to enter voting period. | True |
 | gov.DepositParams.MaxDepositPeriod | 604800000000000 (1 week) | Maximum period for token holders to deposit on a proposal in nanoseconds. | True |
-| gov.VotingParams.VotingPeriod | 604800000000000 (1 week) | Duration of the voting period in nanoseconds. | True |
+| gov.VotingParams.VotingPeriod | 1209600000000000 (2 weeks) | Duration of the voting period in nanoseconds. | True |
```

Comment:
This line shouldn't be modified

```suggestion
| gov.VotingParams.VotingPeriod | 604800000000000 (1 week) | Duration of the voting period in nanoseconds. | True |
```

Reply:
I had some conflict issues and while working on them, I think this issue popped up!
Will make changes to them